src/scenes/Preloader.ts

```diff
@@ -33,6 +35,26 @@ export class Preloader extends Scene
         this.load.setPath('assets');
 
         this.load.image('logo', 'logo.png');
+
+        this.load.image('play-icon', 'play-icon.png');
+        
+        // Test Only- replace with actual duck spritesheet
+        this.load.image('duck', 'duck.png');
+
+        // Load custom font
+        this.load.font(
+        'PixelGame',
+        'https://raw.githubusercontent.com/google/fonts/refs/heads/main/ofl/pressstart2p/PressStart2P-Regular.ttf',
+        'truetype'
+        );
+
+        this.load.font(
```

Comment:
General practice for javascript is to indent function arguments by one level for readability of code

```suggestion
        this.load.font(
            'Kenney-Future-Narrow',
            'assets/fonts/Kenney-Future-Narrow.ttf',
            'truetype'
        );
```

Reply:
Thanks for the suggestion! I have made the recommended changes